src/niscope/metadata/enums_addon.py

```diff
@@ -139,38 +139,810 @@
             {
                 'name': 'AC',
                 'value': 0,
-'documentation': {
-'description': 'AC coupling',
-},
+                'documentation': {
+                    'description': 'AC coupling',
+                },
             },
             {
                 'name': 'DC',
                 'value': 1,
-'documentation': {
-'description': 'DC coupling',
-},
+                'documentation': {
+                    'description': 'DC coupling',
+                },
             },
             {
                 'name': 'HF_REJECT',
                 'value': 2,
-'documentation': {
-'description': 'Highpass filter coupling',
-},
+                'documentation': {
+                    'description': 'Highpass filter coupling',
+                },
             },
             {
                 'name': 'LF_REJECT',
                 'value': 3,
-'documentation': {
-'description': 'Lowpass filter coupling',
-},
+                'documentation': {
+                    'description': 'Lowpass filter coupling',
+                },
             },
             {
                 'name': 'AC_PLUS_HF_REJECT',
                 'value': 1001,
-'documentation': {
-'description': 'Highpass and lowpass filter coupling',
-},
+                'documentation': {
+                    'description': 'Highpass and lowpass filter coupling',
+                },
             },
         ],
     },
-}
+    'ArrayMeasurement': {
+        'values': [
+            {
+                'name': 'NO_MEASUREMENT',
```

Comment:
All names should use the C name `NISCOPE_VAL_<>`. So this one should be `NISCOPE_VAL_NO_MEASUREMENT`